third_party/CMakeLists.txt

```diff
@@ -27,3 +27,6 @@
 #
 
 add_subdirectory(openthread)
+add_subdirectory(cJSON)
```

Comment:
Do we have a CMake file in exactly `third_party/cJSON`?

Reply:
Yes